components/dot_menu/dot_menu.tsx

```diff
@@ -26,73 +25,79 @@ import DotsHorizontalIcon from 'components/widgets/icons/dots_horizontal';
 const MENU_BOTTOM_MARGIN = 80;
 
 export const PLUGGABLE_COMPONENT = 'PostDropdownMenuItem';
+type Props = {
+    post: any;
+    teamId?: string;
+    location?: 'CENTER' | 'RHS_ROOT' | 'RHS_COMMENT' | 'SEARCH' | string; // TechDebt: Made non-mandatory while converting to typescript
+    commentCount?: number;
+    isFlagged?: boolean;
+    handleCommentClick?: React.EventHandler<React.MouseEvent>;
+    handleDropdownOpened?: (open: boolean) => void;
+    handleAddReactionClick?: () => void;
+    isMenuOpen?: boolean,
+    isReadOnly: boolean | null,
+    pluginMenuItems: any[];
+    isLicensed?: boolean, // TechDebt: Made non-mandatory while converting to typescript
+    postEditTimeLimit?: string, // TechDebt: Made non-mandatory while converting to typescript
+    enableEmojiPicker: boolean;
+    channelIsArchived?: boolean, // TechDebt: Made non-mandatory while converting to typescript
+    currentTeamUrl?: string, // TechDebt: Made non-mandatory while converting to typescript
+
+    /**
+     * Components for overriding provided by plugins
+     */
+    components?: any, // TechDebt: Made non-mandatory while converting to typescript
+
+    actions?: {
+
+        /**
+         * Function flag the post
+         */
+        flagPost: (post_id: number) => void;
```

Comment:
Are we sure post_id is `number`? I think it should be string?

Comment applies to all functions definitions in actions.

Reply:
fixed in latest commit